src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -45,23 +56,74 @@ public DriveSubsystem() {
 
     // init drive function
     m_ddrive = new DifferentialDrive(m_left, m_right);
+    // config pid controller for motors.
+    m_turnController.enableContinuousInput(-180.0f, 180.0f);
   }
 
   // default tank drive function
   // **tank drive = specific control style where two parallel forces of motion are controlled to
   // create linear and rotational motion
   public void tankDrive(double leftSpeed, double rightSpeed) {
-    m_ddrive.tankDrive(leftSpeed, rightSpeed);
+    m_ddrive.tankDrive(
+        Constants.MAX_SPEED * leftSpeed,
+        -Constants.MAX_SPEED * rightSpeed); // negative as motors are swapped rn
   }
 
   public void backward() {
-    this.tankDrive(0.5, -0.5);
+    this.tankDrive(0.5, 0.5);
```

Comment:
Where is this function used/where do we plan to use it?

Reply:
used rn by autonomous, will probably still be used for now.

Reply:
Lets move towards using methods directly rather than abstracting parameters away into the drive class, we can use constants if we want to reuse the same values in multiple calls or have defaults.

Reply:
removed.